pkg/virt-controller/services/virtiofs.go

```diff
@@ -75,39 +75,22 @@ func isAutoMount(volume *v1.Volume) bool {
 	return volume.ServiceAccount != nil
 }
 
-func virtioFSMountPoint(volume *v1.Volume) string {
-	volumeMountPoint := fmt.Sprintf("/%s", volume.Name)
-
-	if volume.ConfigMap != nil {
-		volumeMountPoint = config.GetConfigMapSourcePath(volume.Name)
-	} else if volume.Secret != nil {
-		volumeMountPoint = config.GetSecretSourcePath(volume.Name)
-	} else if volume.ServiceAccount != nil {
-		volumeMountPoint = config.ServiceAccountSourceDir
-	} else if volume.DownwardAPI != nil {
-		volumeMountPoint = config.GetDownwardAPISourcePath(volume.Name)
-	}
+// virtiofsRequiresExtraVolume returns if the container needs an extra virtiofs volume
+// where the socket for the virtiofs placeholder will be located
+func virtiofsRequiresExtraVolume(vmi *v1.VirtualMachineInstance) bool {
```

Comment:
I'd drop this func and use `virtiofs.HasFilesystemPersistentVolumes(vmi)` directly.

Reply:
Since it's used inside the template, I prefer to keep the knowledge about when/why requires an extra volume in a single place